src/routes/Search/SearchForm.tsx

```diff
@@ -58,19 +58,23 @@ const SearchForm = ({ dataLength }: IProps) => {
 
   const handleSubmit = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault()
+    if (inputRef.current) {
+      inputRef.current.blur()
+    }
     const searchedLog = store.get('searchedLog') || []
 
     if (searchedLog.findIndex((item: IDiseaseDataItem) => item.sickNm === inputValue) === -1) {
+      if (!inputValue.trim()) return
       store.set('searchedLog', [{ sickNm: inputValue }, ...searchedLog].slice(0, 6))
     }
 
     if (!inputValue.trim()) return
-    dispatch(setSearchValue(inputValue))
+    dispatch(setSearchValue(inputValue.split(' ').join('')))
   }
 
   useDebounce(
     () => {
-      !isApiBlocked && dispatch(setSearchValue(inputValue))
+      !isApiBlocked && dispatch(setSearchValue(inputValue.split(' ').join('')))
```

Comment:
```suggestion
      !isApiBlocked && dispatch(setSearchValue(inputValue.replace(/\s+/g, '')))
```

띄어쓰기 없애기 위해서 split 후 join 사용하신거라면 replace 메서드 사용하시는 건 어떨까용
글구 띄어쓰기 없애신 이유가 Fuzzy Matching 알고리즘에서 사용하기 위함인가요 ?